### ISSUE TITLE
propgcc does not build on latest Mac OS X (10.9 Mavericks) binutils build fails

### ISSUE THREAD

**Issue by GoogleCodeExporter** [parallaxinc/propgcc#64]
```
What steps will reproduce the problem?
1. Download latest propgcc sources to Mac OS X Mavericks (10.9)
2. 'source' setenv_macosx to set Mac OS type build
3. 'sudo ./jbuild.sh'

What is the expected output? Expected propgcc to build

What do you see instead? Error building read elf.c in binutils

[code]
../../../propgcc/binutils/binutils/readelf.c:9051:20: error: adding 'int' to a 
string does not append to the string
      [-Werror,-Wstring-plus-int]
    fputs ("     " + n, stdout);
           ~~~~~~~~^~~
../../../propgcc/binutils/binutils/readelf.c:9051:20: note: use array indexing 
to silence this warning
    fputs ("     " + n, stdout);
                   ^
           &       [  ]
1 error generated.
make[4]: *** [readelf.o] Error 1
[/code]


What version of the product are you using? Propgcc version is 
propellergcc_v0_3_5_2255  On what operating system? Mac OS X 10.9


Please provide any additional information below.  A solution for this problem 
has been described at stack overflow.com 
(http://stackoverflow.com/questions/20337422/cross-compiling-binutils-on-os-x). 
Basically, update binutils to a newer version or change one line in readelf.c 
("use: puts (&" "[n], stdout);")

```

Original issue reported on code.google.com by `dgat...@gmail.com` on 31 Dec 2013 at 6:01

Attachments:
- [MacOSXMavericksCompileLog.txt](https://storage.googleapis.com/google-code-attachments/propgcc/issue-64/comment-0/MacOSXMavericksCompileLog.txt)
